pages/_app.js

```diff
@@ -115,6 +120,18 @@ export default function App({ Component, pageProps }) {
     setTitleLabel(event.target.value);
   }
 
+  function handleLineColorChange(event) {
+    setLineColor(event.target.value);
+  }
+
+  function handleLineStyleChange(event) {
+    setLineStyle(event.target.value);
+  }
+
+  function handleLineWidthChange(event) {
+    setLineWidth(event.target.value);
+  }
+
   return (
     <>
```

Comment:
No need for the React Fragment here, as you are only returning 1 element (`Layout`)

Reply:
Done.